src/main/resources/web/app/qs-form.ts

```diff
@@ -123,15 +123,14 @@ export class QsForm extends LitElement {
 
         this._jsonFetch(controller, 'GET', data, this._page > 0 ? this.initialTimeout : this.moreTimeout)
             .then((r: any) => {
-                let event = QS_RESULT_EVENT;
                 if (this._page > 0) {
                     this._currentHitCount += r.hits.length;
                 } else {
                     this._currentHitCount = r.hits.length;
                 }
                 const total = r.total?.lowerBound;
                 const hasMoreHits = r.hits.length > 0 && total > this._currentHitCount;
-                this.dispatchEvent(new CustomEvent(event, {detail: {...r, page: this._page, hasMoreHits}}));
+                this.dispatchEvent(new CustomEvent(QS_RESULT_EVENT, {detail: {...r, search: data, page: this._page, hasMoreHits}}));
```

Comment:
mm I'm probably missing something, but I can't find where the `search: data` is used,

Reply:
ohh 😃 ok makes sense 😃